packages/app-admin-ui/client/pages/Signin.js

```diff
@@ -124,7 +127,7 @@ const SignInPage = () => {
       </Alerts>
       <PageTitle>{siteName}</PageTitle>
       <Form method="post" onSubmit={onSubmit}>
-        <KeystoneLogo />
+        {Branding ? <Branding /> : <KeystoneLogo />}
```

Comment:
with recently added one hook by @MadeByMike and I added other hooks in #2469 the paradigm is that we pass on the existing component and you can decide if you want to render additional components or replace it. 

In this scenario it will mostly be replaced, I am with the design you are proposing. 

I also feel that we should have more hooks for branding, not just `Branding` as logo replacement. There are other places the Logo size may be small and we may need multiple hooks for each logo placement and guideline for size etc.

eventually: 
```js
{
   branding: {
       logo: LogoComponent,
      footer: FooterCompnent,
      primaryColor: ColorCSSObject,
// etc
   }
}
```